forge-gui/res/cardsfolder/upcoming/invert_polarity.txt

```diff
@@ -0,0 +1,8 @@
+Name:Invert Polarity
+ManaCost:U U R
+Types:Instant
+A:SP$ FlipACoin | TargetType$ Spell | TgtZone$ Stack | WinSubAbility$ GainControl | LoseSubAbility$ CounterIt | ValidTgts$ Card | SubAbility$ DBPutCounter | | SpellDescription$ Choose target spell, then flip a coin. If you win the flip, gain control of that spell and you may choose new targets for it. If you lose the flip, counter that spell.
```

Comment:
`SubAbility$ DBPutCounter | | ` all seems copy pasta
This could probably use a StackDescription$ ?